### PR TITLE
small changes

### PR DESCRIPTION
**Pull request by Jeder321**
deleted "form" to make this thing more readable(also that triggered me)

### CODE REVIEW

**Comment by zerebos** [comment]
Was form removed from the DOM? Including the form element allows it to ensure it overrides the default style without an !important

**Comment by pointydev** [comment]
`form` wasn't removed from the DOM, however there are new margin overrides in the default CSS, seems an important may be needed, as the current implementation doesn't seem to be working, see below:

File: code-files/beta/beta.css [hunk 1]
```diff
@@ -14005,7 +14005,7 @@ body .theme-dark #friends .headerBar-cxbhPD .tab-bar .tab-bar-item.tab-bar-item-
 =======
 /* Latest Hotfixes */
 /* form changes */
-form .channelTextArea-os01xC {
+.channelTextArea-os01xC {
     margin: 0;
```
Comment (pointydev) [inline code-files/beta/beta.css:14009]: Would recommend changing to:
```css
.channelTextArea-os01xC {
    margin: 0!important;
}
```

File: code-files/update/version.css [hunk 2]
```diff
@@ -13884,7 +13884,7 @@ body .sp-hidden[style*="top"]{
 
 /* Latest Hotfixes */
 /* form changes */
-form .channelTextArea-os01xC {
+.channelTextArea-os01xC {
     margin: 0;
```
Comment (pointydev) [inline code-files/update/version.css:13888]: Would recommend changing to:
```css
.channelTextArea-os01xC {
    margin: 0!important;
}
```

**Review by pointydev — CHANGES_REQUESTED** [review]

**Comment by zerebos** [comment]
This screenshot looks like you don't have the theme enabled as there should other things being applied to that element from beards theme.

**Comment by pointydev** [comment]
@rauenzi Theme is enabled, however it is via DI custom css.

**Comment by zerebos** [comment]
The CSS doesn't appear in the inspector screenshot. It should at least attempt to be applied even if it doesn't have the highest priority  unless form was removed

**Comment by pointydev** [comment]
Not appearing whatsoever, even with cache disabled.

**Pull request closed without merging**